File: PROYECTO/V1/Zarpas_Backend/src/main/java/com/gestion/modelo/comentario_reaccion.java
package com.gestion.modelo;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Builder;

import java.sql.Timestamp;

@Entity
@Table(name= "comentario_reaccion")
@Builder
public class comentario_reaccion {

    @Id
    @Column(name = "id_comentario")
    private Long id_comentario;
    @Id
    @Column(name = "id_usuario")
    private Long id_usuario;

    @Column(name = "tipo_reaccion")
    private Enum tipo_reaccion;

    @Column(name = "fecha_reaccion")
    private Timestamp fecha_reaccion;

    public comentario_reaccion() {

    }
}

File: PROYECTO/V1/Zarpas_Frontend/gestion-zarpas-frontend/src/app/modelos/ComentarioReaccion.ts
import { TipoReaccion } from "./TipoReaccion";

export interface ComentarioReaccion {
    idUsuario: number;
    idComentario: number;
    tipoReaccion: TipoReaccion;
    fechaReaccion?: Date | null;
  }
  export interface ComentarioReaccionRequest {
    idUsuario: number;
    idComentario: number;
    tipoReaccion: TipoReaccion | null;
  }